mailtrap/models/projects.py

```
@@ -0,0 +1,18 @@
+from pydantic import BaseModel
+from pydantic import Field
```

Ruff (0.12.2)

`pydantic.Field` imported but unused

Remove unused import: `pydantic.Field`

(F401)